apps/penxle.com/src/lib/tiptap/nodes/blockquote.ts

```diff
@@ -17,20 +21,32 @@ export const inputRegex = /^\s*>\s$/;
 
 export const Blockquote = Node.create({
   name: 'blockquote',
-  content: 'prose+',
   group: 'block',
+  content: 'paragraph+',
   defining: true,
 
   addAttributes() {
     return {
       kind: {
-        default: 1,
-        parseHTML: (element) => element.dataset.kind,
-        renderHTML: (attributes: { kind: Kind }) => {
-          const align = attributes.kind === 3 ? { 'data-text-align': 'center' } : {};
+        isRequired: true,
+        parseHTML: (element) => {
+          if (element.dataset.kind === undefined) {
+            return 1;
+          }
 
-          return { 'data-kind': attributes.kind.toString(), ...align };
+          const blockquote = Number.parseInt(element.dataset.kind);
+          return closest(blockquote, blockquotes);
         },
+        renderHTML: ({ kind }) => ({
+          'class': clsx(
+            kind === 1 && 'border-l-0.1875rem border-text-primary pl-0.625rem my-0.34375rem p-r-6',
```

Comment:
```suggestion
          'class': clsx(
            'border-text-primary pl-0.625rem my-0.34375rem'.
            kind === 1 && 'border-l-0.1875rem p-r-6',
```

P3: 공통 스타일은 외부로 뺴는 게 어떨까 싶습니다.